README.md

```diff
@@ -18,10 +16,15 @@ julia> Pkg.add("Missings")
 
 ## Project Status
 
-The package is tested against the current Julia `0.6` release and nightly on Linux, OS X, and Windows.
-
 Starting from Julia `0.7` the `Missing` type and basic functionalities related to it have been merged to core Julia.
-This package still provides additional features to handle `missing` values.
+See [the Julia manual](https://docs.julialang.org/en/v1/manual/missing/index.html) for general documentation on missing values in Julia.
+This package still provides additional features to handle `missing` values:
+- `Missings.T` to extract `T` from a `Union{T, Missing}` type
```

Comment:
Is it worth mentioning this at all given that `Base.nonmissingtype` exists?

Reply:
Until it's exported I'd rather not recommend it. Also it's broken in complex cases in older Julia versions.

Reply:
That's fair I suppose.

Reply:
#102

Reply:
```suggestion
- `Missings.nonmissingtype` to extract `T` from a `Union{T, Missing}` type
```